tensorflow/inference/docker/1.15/py3/Dockerfile.neuron

```diff
@@ -142,12 +142,13 @@ RUN ${PIP} install --no-cache-dir -U \
     grpcio==1.24.1 \
     protobuf==3.10.0 \
     gast==0.2.2 \
-    "awscli<2" \
-    tensorflow_serving_api==1.15.0 --no-deps --extra-index-url https://pip.repos.neuron.amazonaws.com \
-    tensorboard-neuron --extra-index-url https://pip.repos.neuron.amazonaws.com \
-    neuron-cc --extra-index-url https://pip.repos.neuron.amazonaws.com \
-    tensorflow-neuron --extra-index-url https://pip.repos.neuron.amazonaws.com
+    "awscli<2"
 
+RUN ${PIP} install tensorflow_serving_api==1.15.0 --no-deps --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && ${PIP} install tensorboard-neuron --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && ${PIP} install neuron-cc --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && ${PIP} install tensorflow-neuron --extra-index-url https://pip.repos.neuron.amazonaws.com
```

Comment:
Please add a comment before this `RUN` statement explaining that the `pip install` statements have been separated out into multiple sequentially executed statements to resolve package dependencies during installation.